### PR TITLE
Included NULLCON'19, Goa

### PR DESCRIPTION
**Pull request by vamsivd00**
Included NULLCON
1st March, 2019
Holiday Inn Resort, Goa

### CODE REVIEW

**Comment by nikhita** [comment]
@vamsivd00 Can you point me to a link where the conference lists that it supports CFPs and has a Code of Conduct? This repo only accepts conferences which supports CFPs and I'd like to include conferences which support CoC from now on...

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [Pragma IIITA](http://pragmaconf.tech/) | 19 -20 Jan | Indian Insitute of Information Technology, Allahabad | Pragma is premier student design and developers’ conference held at IIIT-Allahabad. Attracting some of the best student developers from across India, Pragma is a 3 day extravaganza featuring a hackathon, workshops and numerous talks. | No
 | [GopherCon](https://gopherconindia.com/) | 18-19 Jan | Hotel Park Regis, Goa | Go conference in India. Go is an open source project developed by a team at Google and many contributors for the open source community. | [Yes]( https://gopherconindia.com/scholarship/) |
 | [JSFoo Pune](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA Trade Tower, Pune | The second Pune edition of India’s premier JavaScript conference. | No |
-
+| [NULLCON Goa](https://nullcon.net/website/) | 01 March | Holiday Inn Resort, Goa | A two day information exchange platform and CTF. | Yes |
```
Comment (nikhita) [inline README.md]: I couldn't find a link to the scholarship. If you could point me to it, that'd be great! :)

Also, the conferences are listed in reverse chronological order as per date.

Comment (vamsivd00) [reply README.md]: This is the link to the CFP : https://nullcon.net/website/goa-2019/cfp.php

The scholarship details are available on the registration page.
https://nullcon.net/website/register.php

Comment (vamsivd00) [reply README.md]: The CoC can be found here.
https://nullcon.net/website/conference-policies.php

Comment (vamsivd00) [reply README.md]: I have updated it. Please check if everything is okay.

**Pull request merged**